boards/imix/src/test/ipv6_lowpan_test.rs

```diff
@@ -469,6 +469,10 @@ impl<'a, A: time::Alarm<'a>> TxClient for LowpanTest<'a, A> {
     }
 }
 
+#[allow(clippy::cast_ptr_alignment)]
+// This test contains non-idiomatic rust that involves casting
+// raw packet buffers to headers, but it works for this platform
+// and is just a test, so we included a clippy allow.
```

Comment:
If it's just a test, do we care about the performance of making a copy instead of an `unsafe` memory transmute? There are reasons why this lint exists, and maybe so far it worked on this platform because the data was already aligned, but maybe it will break on this platform in the future if the data becomes unaligned.

I'd vote for fixing the problem rather than allowing this lint.

Reply:
Fixed, and tested again on Imix.